libsoftwarecontainer/src/gateway/cgroups/cgroupsparser.cpp

```diff
@@ -27,25 +28,94 @@ CGroupsParser::CGroupsParser()
 {
 }
 
-ReturnCode CGroupsParser::parseCGroupsGatewayConfiguration(const json_t *element)
+std::string CGroupsParser::byteRegulation(const std::string settingValue,
+                                                        const int multiply)
+{
+    char *end;
+    long long limit_in_bytes = std::strtoll(settingValue.c_str(), &end, 10);
+    if (errno == ERANGE){
```

Comment:
Yeah space will be there. 
It is not mandatory to check return value, if the converted value falls out of range of a range error occurs and  LLONG_MAX or LLONG_MIN is returned. checking whether it is set to LLONG_MAX is nto so different than checking errno in my opinion.